testsettings.py

```diff
@@ -3,6 +3,8 @@
 import settingshelper as helper
 from settings import *
 
+USING_CITUS = any(db for db in DATABASES.values() if db.get('ROLE') == 'citus_master')
```

Comment:
nit: could be more succinctly phrased as
```py
any(db.get('ROLE') == 'citus_master' for db in DATABASES.values())
```

Reply:
```suggestion
USING_CITUS = any(db.get('ROLE') == 'citus_master' for db in DATABASES.values())
```